sdk/src/main/java/com/uber/sdk/rides/client/model/RideReceipt.java

```diff
@@ -0,0 +1,148 @@
+
+package com.uber.sdk.rides.client.model;
+
+import java.util.List;
+
+import javax.annotation.Nullable;
+
+/**
+ * A receipt for a completed request.
+ * See
+ * <a href="https://developer.uber.com/docs/v1-requests-receipt">Ride Request Receipt</a>
+ * for more information.
+ */
+public class RideReceipt {
+    private String request_id;
+    private List<Charge> charges;
+
+    @Nullable
+    private Charge surge_charge;
+
+    private List<Charge> charge_adjustments;
+    private String normal_fare;
+    private String subtotal;
+    private String total_charged;
+
+    @Nullable
+    private Float total_owed;
+
+    private String currency_code;
+    private String duration;
+    private String distance;
+    private String distance_label;
+
+    /**
+     * The unique ID of the ride.
+     */
+    public String getRequestId() {
```

Comment:
Indeed, using the @SerializedName annotation is better in multiple ways:
 - allows flexibility in terms of field names
 - less error prone, a sigle letter difference in the field name from the JSON response yields a parsing error
 - it makes it easier to generate getters and setters using an IDE